velox/docs/monthly-updates/august-2023.rst

```diff
@@ -0,0 +1,67 @@
+******************
+August 2023 Update
+******************
+
+
+Documentation
+=============
+
+* Add :doc:`adaptive disabling of partial aggregation </develop/aggregations>`.
+* Add :doc:`Aggregate::toIntermediate() API </develop/aggregate-functions>`.
+* Add :doc:`documentation about connectors </develop/connectors>`.
+
+Core Library
+============
+
+* Add support for remote functions. :pr:`6104`
+* Add support for accessing sub-fields by index. :pr:`6253`
+* Add support for DECIMAL in UnsafeRowSerializer.
+* Add checks for verifying internal state of a vector. :pr:`5885`
+* Enhance COALESCE and SWITCH special forms to check all inputs are of same type.
+* Fix exception handling in TRY_CAST for JSON inputs. :pr:`6116`
+* Optimize single partition spilling.
+
+
+Presto Functions
+================
+
+* Add :func:`array_remove`, :func:`date_format`, :func: `format_datetime`, `last_day_of_month`, :func:`split_to_map`, :func:`trim` functions.
+* Add :func:`gamma_cdf`, :func:`poisson_cdf`, :func:`laplace_cdf` .
+* Add :func:`wilson_interval_lower`, :func:`wilson_interval_upper` functions.
+* Optimize :func:`array_constructor` function.
+
+
+Spark Functions
+===============
+
+* Add :spark:func:`date_add`, :spark:func:`date_sub`, :spark:func:`week_of_year`, :spark:func:`dayofyear`, :spark:func:`dayofmonth`, :spark:func:`dayofweek` functions.
+* Add :spark:func:`pmod` function.
+* Add :spark:func:`max_by`, :spark:func:`min_by` aggregate functions.
+* Add :spark:func:`row_number` window function.
+
+
+Hive Connector
+==============
+
+* Add support for writing to `Google Cloud Storage <https://cloud.google.com/storage>`_. :pr:`5685`
+* Fix writing to HDFS as part of CREATE TABLE AS SELECT (CTAS) query. :pr:`5663`
+
+
+Performance and Correctness
+===========================
+
+* Add benchmark for spilling. :pr:`6071`
+* Add protection for out of memory errors for table creation during hash builds.
+* Enhance Aggregation Fuzzer to test abandon partial aggregation code paths.
+
+
+Build System
+============
+
+* Add longer fuzzer runs when pull request has changes to certain files and paths. :pr:`6009`
```

Comment:
Should this go into Performance and Correctness section? It is not about build system, is it?